lib/foreman_discovery/engine.rb

```diff
@@ -9,6 +9,7 @@ module ForemanDiscovery
   class Engine < ::Rails::Engine
 
     # support pre-4 Rails versions
+    config.autoload_paths += Dir["#{config.root}/app/controllers/foreman_discovery/concerns"]
     config.autoload_paths += Dir["#{config.root}/app/controllers/concerns"]
```

Comment:
Why adding another path? We already have that on line 13. But if you prefer this path, just move all the concerns into the new folder.